packages/svelte/src/internal/client/reactivity/deriveds.js

```diff
@@ -323,6 +323,20 @@ export function execute_derived(derived) {
 	return value;
 }
 
+// in process_effects if state is written to in a user effect we reschedule the rest of the
+// tree. However if a derived is updated in an effect we also increase the write version
+// so we need to keep track of how many deriveds were written to in the effect so
+// that we can subtract that from the write version before rescheduling unnnecessarily
+let derived_writes = 0;
+
+export function get_derived_writes() {
+	return derived_writes;
+}
```

Comment:
FWIW we don't need to expose function wrappers to get values from different modules, only to set them. we can just `export let derived_writes`

Reply:
I'm always so skeptical of live exports...like under my skin I know they works but I just can't trust myself enough to do it 😂